app/controllers/admin/dashboard_controller.rb

```diff
@@ -2,6 +2,7 @@ module Admin
   class DashboardController < Admin::BaseController
     def show
       @active_flags_count = Flag.count_for(:active)
+      @active_announcements_count = Announcement.active_count
```

Comment:
Nit: I'd be tempted to inline the count and chain it off the active scope:

```suggestion
      @active_announcements_count = Announcement.active.count
```

The most reusable scopes wrap up complex complex queries or filters queries like our active / expired scopes. Things like counts are usually best just being chained to those. Thats definitely not a hard and fast rule though! lol